### PR TITLE
Feature/complete aboutpage

### PR DESCRIPTION
**Pull request by muhammadqamar**
## Summary

updated home and about us page 

## Quality Checklist

- [ ] New features are fully tested (No matter how much Coverage Karma you have)
- [ ] **[v0.2 onward]** New features are included in the documentation
- [ ] **[v0.2 onward]** Breaking changes are documented with clear instructions of what 

### Coverage Karma

- [ ] If your PR decreases test coverage, do you feel you have built enough `Coverage Karma`* to justify it?

*Coverage Karma can be earned by disproportionally increasing test coverage in the rest of your contributions.
It's an honesty policy - you keep count of your own.

### CODE REVIEW

**Review by thclark — CHANGES_REQUESTED** [review]
@muhammadqamar please find attached a review of suggested changes, mostly extremely good, thank you - the majority of the changes are extremely minor.

I've realised that I made a mistake and didn't push some code that wired up the font correctly, and also contains a couple of points where I override the typography in the theme, there are a couple of comments where I suggest that so if you pull `feature/new-design` and merge it into your current branch, I think you'll find that helpful.

File: src/components/hexagon/HexagonSvg.jsx [hunk 1]
```diff
@@ -9,28 +9,47 @@ const useStyles = makeStyles((theme) => ({
     display: 'flex',
     alignItems: 'center',
     justifyContent: 'center',
+    margin: '1rem',
   },
-}))
-const HexagonSvg = ({ svg }) => {
-  const classes = useStyles()
-  const customStyle = {
+  hexagonMB: {
```
[the remark this answers is not in the document]
Comment (muhammadqamar) [reply src/components/hexagon/HexagonSvg.jsx:14]: I have expanded the word “MB” to full name (change only this page. Should i change in every pages).
on some places I have use this word which is mean “For Mobile View”

File: src/components/sectionelements/sectionwithhexagon/HexagonSection.jsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    backgroundColor: `#D31020`,
+  },
+  btnBox: {
+    // marginTop: '24px',
+  },
+  titleBox: {
+    marginBottom: '10px',
+  },
+}))
+const HexagonSection = () => {
```
[the remark this answers is not in the document]
Comment (muhammadqamar) [reply src/components/sectionelements/sectionwithhexagon/HexagonSection.jsx:76]: Changed the component name (Covert "HexagonSection" to "MianPageHexagonSection" )

File: src/components/sectionelements/sectionwithhexagon/AboutHexagonSection.jsx [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+                  its work on digital twins.
+                </Typography>
+              </Box>
+              <Box display="flex" pb={4}>
+                <Typography
+                  variant="subtitle1"
+                  component="h2"
+                  className={classes.description}
+                >
+                  OIn 2021, Octue became a non-profit dedicated to developing
```
[the remark this answers is not in the document]
Comment (muhammadqamar) [reply src/components/sectionelements/sectionwithhexagon/AboutHexagonSection.jsx:134]: Removed the First Character

File: src/components/hexagon/HexagonSvg.jsx [hunk 4]
```diff
@@ -9,28 +9,47 @@ const useStyles = makeStyles((theme) => ({
...
+}))
+const HexagonSvg = ({ svg }) => {
+  const classes = useStyles()
+  // const customStyle = {
+  //   clipPath: 'polygon(50% 0, 100% 25%, 100% 75%, 50% 100%, 0 75%, 0 25%)',
+  //   backgroundColor: '#1895EF',
+  //   height: '120px',
+  //   width: '105px',
+  //   marginLeft: '2px',
+  // }
```
[the remark this answers is not in the document]
Comment (muhammadqamar) [reply src/components/hexagon/HexagonSvg.jsx:40]: I have Removed these comments

File: src/components/hexagon/HexagonSvg.jsx [hunk 5]
```diff
@@ -9,28 +9,47 @@ const useStyles = makeStyles((theme) => ({
...
       </Box>
     </>
   )
 }
-HexagonSvg.defaultProps = {
-  svg: BuildSvg,
-}
+// HexagonSvg.defaultProps = {
+//   svg: BuildSvg,
+// }
```
[the remark this answers is not in the document]
Comment (muhammadqamar) [reply src/components/hexagon/HexagonSvg.jsx:54]: Reintroduce propTypes and DefaultProps has been Done

File: src/components/hexagon/MultipleHexagonWithImage.jsx [hunk 6]
```diff
@@ -10,22 +10,29 @@ const useStyles = makeStyles((theme) => ({
     display: 'flex',
     marginLeft: '175px',
     marginTop: '-30px',
+
```
[the remark this answers is not in the document]
Comment (muhammadqamar) [reply src/components/hexagon/MultipleHexagonWithImage.jsx:13]: Removed extra line

File: src/components/hexagon/MultipleHexagonWithImage.jsx [hunk 7]
```diff
@@ -10,22 +10,29 @@ const useStyles = makeStyles((theme) => ({
...
 }))
-const MultipleHexagonWithImage = () => {
+const MultipleHexagonWithImage = ({ image }) => {
   const classes = useStyles()
   return (
     <>
       <Box className={classes.boxStyle}>
-        <SimpleHexagon bgHexagomImg={Img1} bgX="10%" bgY="20%" />
-        <SimpleHexagon bgHexagomImg={Img1} bgX="20%" bgY="30%" />
+        <SimpleHexagon bgHexagomImg={image} bgX="10%" bgY="20%" />
```
[the remark this answers is not in the document]
Comment (muhammadqamar) [reply src/components/hexagon/MultipleHexagonWithImage.jsx:25]: Changed the name to "bgHexImg"

File: src/components/hexagon/SimpleSmallHexagon.jsx [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+import BackgroundImage from '../../assets/images/hexagon/Hero.png'
+import React from 'react'
+const useStyles = makeStyles((theme) => ({
+  hexagonShapeParent: {
+    clipPath: 'polygon(50% 0, 100% 25%, 100% 75%, 50% 100%, 0 75%, 0 25%)',
+    backgroundColor: '#fff',
+    height: '80px',
+    width: '70px',
+    marginLeft: '2px',
+    // padding: '6px',
```
[the remark this answers is not in the document]
Comment (muhammadqamar) [reply src/components/hexagon/SimpleSmallHexagon.jsx:11]: Cleaned the comments

File: src/components/hexagon/SimpleSmallHexagon.jsx [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+            }}
+          ></Box>
+        </Box>
+      </Box>
+    </>
+  )
+}
+
+SimpleSmallHexagon.defaultProps = {
+  bgHexagomImg: BackgroundImage,
```
[the remark this answers is not in the document]
Comment (muhammadqamar) [reply src/components/hexagon/SimpleSmallHexagon.jsx:54]: Solved

File: src/components/sectionelements/SectionDetailW3.jsx [hunk 10]
```diff
@@ -0,0 +1,99 @@
...
+    color: 'white',
+    fontSize: '18px',
+    fontWeight: 400,
+    lineHeight: '24px',
+    marginRight: '16px',
+    marginTop: '10px',
+  },
+}))
+
+const SectionDetailW3 = ({ headingOne, headingTwo, description }) => {
```
[the remark this answers is not in the document]
Comment (muhammadqamar) [reply src/components/sectionelements/SectionDetailW3.jsx:58]: This Function Name "SectionDetailW3" has been Converted to "SectionDetailWithTwoHeading"

File: src/components/sectionelements/SectionElement.stories.jsx [hunk 11]
```diff
@@ -0,0 +1,94 @@
...
+import HexagonSection from './sectionwithhexagon/HexagonSection'
+import SvgHexagonSection from './singlehexagonsection/SvgHexagonSection'
+import BackgroundImg from './BackgroundImg'
+import HeroSection from './herosection/HeroSection'
+import HelpSection from './helpsection/HelpSection'
+import JoinSection from './joinsection/JoinSection'
+import PartnerLogo from './PartnerLogo'
+import AerosenseWhite from '../../assets/images/logos/partners/aerosense-white.svg'
+import PartnerSection from './partnersection/PartnerSection'
+export default {
```
[the remark this answers is not in the document]
Comment (muhammadqamar) [reply src/components/sectionelements/SectionElement.stories.jsx:13]: Solved

File: src/components/sectionelements/SectionElement.stories.jsx [hunk 12]
```diff
@@ -0,0 +1,94 @@
...
+
+export const sectionHeading = Template.bind({})
+sectionHeading.args = {}
+
+export const SectionWithDetail = () => {
+  return (
+    <>
+      <SectionDetail
+        heading="You don't need to be a materials scientist to build a LEGO® model.
+"
```
[the remark this answers is not in the document]
Comment (muhammadqamar) [reply src/components/sectionelements/SectionElement.stories.jsx:33]: Removed

File: src/components/sectionelements/sectionwithhexagon/HexagonSection.jsx [hunk 13]
```diff
@@ -0,0 +1,144 @@
...
+            <Box className={classes.hexagonBox}>
+              <MultipleHexagonWithImage image={Image} />
+            </Box>
+          </Box>
+          <Box className={classes.detailBox}>
+            {/* <SectionDetail
+              heading="You don't need to be a materials scientist to build a LEGO® model."
+              description="So why should someone modelling climate or energy systems need to be an expert coder? Yet scientists waste 95% of their day [Source:McKinsey] being general programmers, API architects and DevOps engineers. Our solution: a framework of open-source tools to create, connect, exchange and reuse analysis building blocks."
+              button="The mission"
+            /> */}
```
[the remark this answers is not in the document]
Comment (muhammadqamar) [reply src/components/sectionelements/sectionwithhexagon/HexagonSection.jsx:96]: Removed

File: src/legacy/fields/fieldsStyle.jsx [hunk 14]
```diff
@@ -18,7 +18,8 @@ const fieldsStyle = {
     '&, &event': {
       color: '#3C4858',
       fontWeight: '700',
-      fontFamily: '"Roboto Slab", "Times New Roman", serif',
+      // fontFamily: '"Roboto Slab", "Times New Roman", serif',
```
[the remark this answers is not in the document]
Comment (muhammadqamar) [reply src/legacy/fields/fieldsStyle.jsx:21]: Removed

File: src/components/sectionelements/SectionHeading.jsx [hunk 15]
```diff
@@ -0,0 +1,32 @@
...
+    fontWeight: 400,
+    lineHeight: '56px',
+  },
+}))
+const SectionHeading = ({ text }) => {
+  const classes = useStyles()
+  return (
+    <>
+      <Box>
+        <Typography variant="h1" component="h1" className={classes.title}>
```
[the remark this answers is not in the document]
Comment (muhammadqamar) [reply src/components/sectionelements/SectionHeading.jsx:20]: Component='h1' changed to Component='h2'

File: package.json [hunk 16]
```diff
@@ -55,15 +55,17 @@
...
   },
   "scripts": {
     "build": "gatsby build",
     "clean": "gatsby clean",
     "serve": "gatsby serve",
     "start": "gatsby develop",
     "storybook": "start-storybook -s ./src/assets -p 6006",
-    "build-storybook": "build-storybook -s ./src/assets"
+    "build-storybook": "build-storybook -s ./src/assets",
+    "release": "standard-version"
```
[the remark this answers is not in the document]
Comment (muhammadqamar) [reply package.json:68]: Removed "release": "standard-version"

File: package.json [hunk 17]
```diff
@@ -55,15 +55,17 @@
     "axios-mock-adapter": "^1.19.0",
     "babel-loader": "^8.2.2",
     "chromatic": "^5.7.0",
-    "prettier": "2.2.1"
+    "prettier": "2.2.1",
+    "standard-version": "^9.3.1"
```
[the remark this answers is not in the document]
Comment (muhammadqamar) [reply package.json:59]: Removed standard-version

File: src/components/hexagon/SimpleHexagon.jsx [hunk 18]
```diff
@@ -19,21 +23,28 @@ const useStyles = makeStyles((theme) => ({
...
+      height: '80px',
+      width: '75px',
+    },
   },
 }))
-const SimpleHexagon = ({ bgHexagomImg, bgX, bgY }) => {
+const SimpleHexagon = ({ bgHexagomImg, bgX, bgY, customStyle }) => {
   const classes = useStyles()
   return (
     <>
```
[the remark this answers is not in the document]
Comment (muhammadqamar) [reply src/components/hexagon/SimpleHexagon.jsx:36]: Removed unnecessary fragments

**Comment by thclark** [comment]
Hi @muhammadqamar there's still a couple of unresolved issues here outstanding before I can pull into main, can you fix these up before progressing on other PRs please? That way I can look at other PRS without all these contents in.
Thanks!

File: package.json [hunk 19]
```diff
@@ -2,7 +2,7 @@
   "name": "planex-site",
   "private": true,
   "description": "The octue website frontend",
-  "version": "0.1.1",
+  "version": "0.1.6",
```
[the remark this answers is not in the document]
Comment (muhammadqamar) [reply package.json]: Reverted to version": "0.1.1

File: src/components/hexagon/HexagonSvg.jsx [hunk 20]
```diff
@@ -9,28 +9,47 @@ const useStyles = makeStyles((theme) => ({
...
+  //   width: '105px',
+  //   marginLeft: '2px',
+  // }
 
   return (
     <>
-      <Box style={customStyle} className={classes.wrapIcon}>
-        <img src={svg} alt="" />
+      {/* style={customStyle} */}
+      <Box className={`${classes.wrapIcon} ${classes.hexagonMB}`}>
```
[the remark this answers is not in the document]
Comment (muhammadqamar) [reply src/components/hexagon/HexagonSvg.jsx]: Done

File: CHANGELOG.md [hunk 21]
```diff
@@ -0,0 +1,55 @@
+# Changelog
```
[the remark this answers is not in the document]
Comment (muhammadqamar) [reply CHANGELOG.md:1]: Removed

File: src/components/sectionelements/herosection/HeroSection.jsx [hunk 22]
```diff
@@ -0,0 +1,99 @@
...
+    fontSize: '80px',
+    fontWeight: 400,
+    lineHeight: '88px',
+  },
+  description: {
+    color: theme.palette.text.secondary,
+    fontSize: '24px',
+    fontWeight: 400,
+    lineHeight: '30px',
+    // whiteSpace: 'pre-line',
```
[the remark this answers is not in the document]
Comment (thclark) [reply src/components/sectionelements/herosection/HeroSection.jsx:30]: It needs to be one of the recognised [mui typography variants](https://material-ui.com/components/typography/#component) ‘subtitle’ isn’t one, so the classes won’t get picked up.

[the remark this answers is not in the document]
Comment (thclark) [reply src/components/sectionelements/herosection/HeroSection.jsx:30]: Looks like body2 and subtitle1 are both available ([see here](https://github.com/octue/planex-site/blob/eed6568dde6ef2958c94e5d443429cc028c25b29/src/themes/_base.js#L106))

[the remark this answers is not in the document]
Comment (muhammadqamar) [reply src/components/sectionelements/herosection/HeroSection.jsx:30]: But the property mentioned in theme were not matching according to figma design.

[the remark this answers is not in the document]
Comment (thclark) [reply src/components/sectionelements/herosection/HeroSection.jsx:30]: @muhammadqamar hang on, let me show you what I mean. Give me 15 mins!

File: src/components/sectionelements/herosection/HeroSection.jsx [hunk 23]
```diff
@@ -0,0 +1,99 @@
...
+    height: '100%',
+    width: '100%',
+    background: 'linear-gradient(0deg, #212121 0%, rgba(30, 34, 34, 0) 100%)',
+  },
+  title: {
+    display: 'flex',
+    color: theme.palette.text.primary,
+    fontSize: '80px',
+    fontWeight: 400,
+    lineHeight: '88px',
```
[the remark this answers is not in the document]
Comment (thclark) [reply src/components/sectionelements/herosection/HeroSection.jsx:23]: Covered in #63 so will resolve just now

File: src/components/sectionelements/herosection/HeroSection.jsx (hunk 22 above)
[the remark this answers is not in the document]
Comment (thclark) [reply src/components/sectionelements/herosection/HeroSection.jsx:30]: Covered in #63 so will resolve just now

File: CHANGELOG.md (hunk 21 above)
[the remark this answers is not in the document]
Comment (thclark) [reply CHANGELOG.md]: It didn't get removed; done myself in #16a9d13

**Pull request merged**